be/src/olap/tablet_manager.cpp

```diff
@@ -636,6 +636,13 @@ TabletSharedPtr TabletManager::find_best_tablet_to_compaction(
                            << ", tablet_id=" << tablet_ptr->tablet_id();
                 continue;
             }
+            if (compaction_type == CompactionType::BASE_COMPACTION &&
```

Comment:
We should make sure that the rowset is large enough to not perform base comapction. Instead of simply prohibiting base compaction of duplicate tables. Otherwise we might end up with a lot of small files.

Reply:
i'll fix it, hanlde it in pick rowset, filter big rowset if no delete exist